test/Datadog.Trace.Tests/Sampling/RateLimiterTests.cs

```diff
@@ -139,7 +139,9 @@ private static RateLimitResult RunTest(int? intervalLimit, RateLimitLoadTest tes
 
             var limiter = new RateLimiter(maxTracesPerInterval: intervalLimit);
 
-            var traceContext = new TraceContext(Tracer.Instance);
+            var tracerInstance = new Tracer();
+
+            var traceContext = new TraceContext(tracerInstance);
```

Comment:
We're only testing `RateLimiter.Allowed(Span)`, so I think we can use a mock `ITraceContext` instead of a real `Tracer` or `TraceContext`.

Reply:
Can we make a note for the future on this? If this is the fix, I'd rather not spend the cycles on a refactor right now.

Reply:
It's just
```csharp
var traceContext = new Mock<ITraceContext>();
```

and then pass `traceContext.Object` in `new Span(...)` below.

Reply:
Never mind. This would break other tests in this file which rely on `TraceContext.UtcNow` (unless we mocked that property, but that's just making things more complicated than we need).

I posted [an alternative PR](https://github.com/DataDog/dd-trace-dotnet/pull/1047) the removes the `TraceContext` altogether.